lib/querystring.js

```diff
@@ -115,11 +116,6 @@ function qsUnescape(s, decodeSpaces) {
   }
 }
 
-
-const hexTable = [];
-for (var i = 0; i < 256; ++i)
-  hexTable[i] = '%' + ((i < 16 ? '0' : '') + i.toString(16)).toUpperCase();
-
 // These characters do not need escaping when generating query strings:
```

Comment:
I only moved this array because it is used in multiple places (querystring and url). The only other entity in querystring that is somewhat shared is `ParsedQueryString`, a copy of which is in url (as `ParsedQueryString` as well) and in internal/url (as `StorageObject`). I've moved this as well. (There is also an analog of it in events as `EventHandlers`, but I kept it as it is since it'd be weird for that module to import from `internal/querystring`).